### PR TITLE
[section/VaultMaxi] added first version from maxi guide

### PR DESCRIPTION
**Pull request by kuegi**
still WIP, but created the PR to get first feedback

### CODE REVIEW

**Comment by 0ptim** [comment]
Everything looks fine except a problem with all `<ABC>` occurrences. On `npm run start`, the build fails because of them.

You could wrap them with: `

**Comment by kuegi** [comment]
thx, will do

**Comment by kuegi** [comment]
I think its good enough for a first version. Will split into sections, add pics etc. in a next iteration

File: docs/auto/Vault_maxi.md [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+
+## Pitfalls and other things to consider
+
+* only one vault-maxi instance should run on an address to avoid bots interfering with each other. Means that one address can only have one managed vault. If you want to create multiple bots for multiple vaults, every vault needs a distinct address.
+* don’t forget to set up and test telegram notifications. As the bot will warn you if misconfigured or configuration doesn’t provide enough safety.
+* After applying an update don’t forget to check your setup again, to ensure nothing broke and vault is still managed
+* do not use weak security on your AWS account, treat it like your wallet private key. Look up best practices for securing your AWS account + IAM users as good as possible
+* do not just change your LM token without paying back loans of your old token. Please take a look at our FAQ: How can I switch the used dToken for LM?
+* do not set up way 1 and directly afterwards go through way 2 steps. Those are two completely different approaches and will conflict with each other.
+* skipping steps on telegram setup or way 2 steps. If you are skipping one step this might prevent you from running vault-maxi successfully.
```
Comment (Krysh90) [inline docs/auto/Vault_maxi.md]: I guess way 1 and way 2 don't make any sense in the context of this site.
Maybe we should use way 1 as cloud formation template and way 2 as doing it manually

File: docs/auto/Vault_maxi.md [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+
+
+## Security and best practises
+* on your AWS root account, make sure to use a long password, which is nowhere written down or saved into a file. There are multiple “how to create a strong password”, or use a password manager, which can generate and save really long passwords. Using a password manager in general is a great idea. This shifts the strong password problem to unlocking the password manager itself. Your choice
+* Make sure to enable MFA (multi factor authentication) or on other sites referred to as 2FA (two factor authentication). Which generates a 6 digit code that changes every 30 seconds. Means an attacker would need your password and your MFA-device (most likely your phone). How to activate MFA?
```
Comment (Krysh90) [inline docs/auto/Vault_maxi.md]: How to activate MFA should be linked to the explanation or removed

**Comment by 0ptim** [comment]
Congrats 💯
https://www.defichainwiki.com/docs/auto/Vault_maxi

**Comment by 0ptim** [comment]
Search also working
image

**Pull request merged**